projects/sartography-workflow-lib/src/lib/modules/pipes/to-formly.pipe.ts

```diff
@@ -147,66 +148,63 @@ export class ToFormlyPipe implements PipeTransform {
-          resultField.defaultValue = field.default_value;
+          this.setDefaultValue(model, resultField, field, def);
           break;
         case 'textarea':
           resultField.type = 'textarea';
-          resultField.defaultValue = field.default_value;
+          this.setDefaultValue(model, resultField, field, def);
           resultField.templateOptions.rows = 5;
           break;
         case 'long':
           resultField.type = 'input';
           resultField.templateOptions.type = 'number';
-          resultField.defaultValue = parseInt(field.default_value, 10);
+          this.setDefaultValue(model, resultField, field, def);
           resultField.validators = {validation: ['number']};
           break;
         case 'url':
           resultField.type = 'input';
           resultField.templateOptions.type = 'url';
-          resultField.defaultValue = field.default_value;
+          this.setDefaultValue(model, resultField, field, def);
           resultField.validators = {validation: ['url']};
           break;
         case 'email':
           resultField.type = 'input';
           resultField.templateOptions.type = 'email';
-          resultField.defaultValue = field.default_value;
+          this.setDefaultValue(model, resultField, field, def);
           resultField.validators = {validation: ['email']};
           break;
         case 'tel':
           resultField.type = 'input';
           resultField.templateOptions.type = 'tel';
-          resultField.defaultValue = field.default_value;
+          this.setDefaultValue(model, resultField, field, def);
           resultField.validators = {validation: ['phone']};
           break;
         case 'boolean':
           if (field.properties.find(x => x.id === 'boolean_type' && x.value === 'checkbox')) {
             resultField.type = 'checkbox';
-            resultField.defaultValue = false;
+            this.setDefaultValue(model, resultField, field, def);
             resultField.templateOptions = { indeterminate: false };
             resultField.validators = {validation: ['checked']};
             break;
           }
           else if (!field.properties.find(x => x.id === 'boolean_type')) {
             resultField.type = 'radio';
-            if (field.default_value !== undefined && field.default_value !== null && field.default_value !== '') {
-              resultField.defaultValue = this._stringToBool(field.default_value);
-            }
             resultField.templateOptions.options = [
               {value: true, label: 'Yes'},
               {value: false, label: 'No'},
             ];
+            // If you want a default value set, you have to find it in the options.
+            if (field.default_value) {
+              resultField.defaultValue = resultField.templateOptions.options.find(o => o.value === field.default_value);
+              resultField.expressionProperties['model.' + field.id] = this.getPythonEvalFunction(field, def);
+            }
```

Comment:
Does this mean that default value is NOT evaluated if we are dealing with an enumeration?  Shouldn't we evaluate it here too?